src/services/heroes.js

```diff
@@ -0,0 +1,56 @@
+const heroes = [{
```

Comment:
this isn't a service, should be somewhere else

Reply:
Moved to temp folder, thank you.